src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -245,7 +253,7 @@ class DiskImageEditor extends Component {
   }
 
   isFormValid () {
-    const isNotAllFieldFilled = Object.values(this.state.values).reduce((acc, value) => !value ? true : acc, false)
+    const isNotAllFieldFilled = Object.values(this.state.values).reduce((acc, value) => !(value !== undefined) ? true : acc, false)
```

Comment:
I'm not sure about this condition `!(value !== undefined)`, maybe will be better to rewrite it like that `value === undefined`?

Reply:
Agree with @bond95, this is awkward.  Probably doing explicit checks per field would be easier even if more verbose.

For example:
```jsx
isFormValid () {
  const isErrors = Object.values(this.state.errors).reduce((acc, value) => value ? true : acc, false)
  if (isErrors) {
    return false
  }

  // make sure that required fields have data
  const { values } = this.state
  return this.props.disk
    // edit mode needs: a change, an alias and a disk size
    ? this.changesMade && !!values.alias && values.size > 0 
    // create mode needs: name, size, storage domain, disk type
    : values.alias && values.size > 0 && values.storageDomain && values.format
}
```